TelemetrySite/server/Context/context.py

```diff
@@ -0,0 +1,135 @@
+from flask import request, jsonify
+from flask.views import MethodView
+import utils
+
+
+class Context(MethodView):
+    configNames = ("bmsConfig", "imuConfig", "tmuConfig", "tmsConfig", "pvcConfig", "mcConfig")
+    configShort = ("bms", "imu", "tmu", "tms", "pvc", "mc")
+
+    ## Get all the past config data saved and return 
+    ## any of them with a name
+    #
+    # @return all saved past files
+    def get(self):
+        #declare dictionary for returning data
+        configData={}
+        #loop through each table and pull the names
+        #0 used as a place holder, no args needed
+        #can't use the %s for table names
+       
+        for i in range(0, len(self.configNames)):
+            sqlCommand = f"SELECT configName FROM {self.configNames[i]}"
+            configData[self.configShort[i]] = utils.exec_get_all(sqlCommand, [0,])  
+
+        configData["bike"]=utils.exec_get_all("SELECT configName FROM BikeConfig WHERE configName IS NOT NULL", [0,])
+     
+        return jsonify(configData).get_data(as_text=True), 200
+
+    def post(self):
```

Comment:
I'm probably fine leaving it like this, but I would like some explanation for why you think it `post` should be a member function. To me, it seems like it would make more sense to have all the SQL in a single file instead of sprinkled across multiple, but I haven't thought through the whole design. Why do you want it this way? Or are you planning to change it later?